builtin/plugins/dposv3/dpos.go

```diff
@@ -877,6 +884,44 @@ func (c *DPOS) RegisterCandidate(ctx contract.Context, req *RegisterCandidateReq
 	return c.emitCandidateRegistersEvent(ctx, candidateAddress.MarshalPB(), req.Fee)
 }
 
+func (c *DPOS) Unjail(ctx contract.Context, req *UnjailRequest) error {
+	if !ctx.FeatureEnabled(loomchain.DPOSVersion3_3, false) {
+		return errors.New("DPOS v3.3 is not enabled")
+	}
+	ctx.Logger().Info("DPOSv3 Unjail", "request", req)
```

Comment:
Maybe move this down a bit, just before `statistic.Jailed = false`, don't really want invalid requests spamming the logs

Reply:
Done